integration_tests/dbt_project.yml

```diff
@@ -9,6 +9,8 @@ models:
   # +schema: "hubspot_{{ var('directed_schema','dev') }}" ## To be used for validation testing
 
 vars:
+  # jsons and arrays. reminder to myself to remove source_relation + surrogate keys
+  consistency_test_exclude: ['source_relation', 'id', 'ticket_day_id', 'merged_deal_ids', 'browser', 'contact_ids', 'deal_ids', 'company_ids', 'deal_names', 'contact_emails', 'company_names', 'bcc_emails', 'cc_emails', 'reply_to_email', 'geo_location', 'subscriptions']
```

Comment:
Let's only keep here the columns that will need to be permanently removed. Anything (like `source_relation`) that was only just added should be removed from this list and only used for local testing as it should be included in the next version test.